src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
```

Comment:
It's a redundant comment. Your code has to be readable without comments. We can add comments(especially I would recommend using Java docs for it: https://www.baeldung.com/javadoc) to complicated logic.